configuration-reference/table.md

```diff
@@ -71,7 +71,7 @@ Find details on configuring routing [here](https://docs.pinot.apache.org/operato
-| createInvertedIndexDuringSegmentGeneration | Boolean to indicate whether to create inverted indexes during the segment creation. By default, false i.e. inverted indexes are created when the segments are loaded on the server                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| createInvertedIndexDuringSegmentGeneration | Boolean to indicate whether to create inverted indexes during the segment creation. By default, false i.e. inverted indexes are created when the segments are loaded on the server. Learn more about this setting in [advanced Pinot setup](../developers/advanced/advanced-pinot-setup.md#add-an-inverted-index-to-your-table-automatically).                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
I only added the sentence with the link. If I started editing the text of the cell, I would find myself editing the text across the entire page. That's a big job for another day.